webapp/src/assets/strings.ts

```diff
@@ -0,0 +1,8 @@
+export const COPY = {
+    STEPWISE_RESULT_NOT_FOUND_REGEX: /(Result not found, review _stepsTaken to see what happened\.)\s+(\[{.*}])/g,
+    CHAT_DELETED_MESSAGE: (chatName?: string) =>
+        `Chat ${
+            chatName ? `{${chatName}} ` : ''
+        }has been deleted by another user. Please save any resources you need and refresh the page.`,
```

Comment:
I wonder what resources they can save. The chat has been deleted at this point from the backend.

Reply:
If they want to save message content, screenshots, plan jsons, etc. 

I had it automatically delete chats for all users once the deletion was successful, and just having the chat removed when you're not the initiator was a bit jarring, so I wanted to give the chances to interact with it and refresh on their own accord.

Reply:
I see. I think the resource is a bit misleading. It makes me think maybe I can save something to my account to download something to my local computer.

nit: change it to something like: ...has been deleted by another user. You can still view the most recent chat history at this point. The content will disappear once you refresh or close the app.

Reply:
Changed!